ratatui-widgets/src/scrollbar.rs

```diff
@@ -451,7 +471,11 @@ impl ScrollbarState {
         self
     }
 
-    /// Sets the items' size.
+    /// Sets the viewport size.
+    ///
+    /// This needs to be set if the viewport size differs
+    /// from the track length, which may be the case if begin
+    /// and end symbols are set.
```

Comment:
Can you expand on what you're saying with this a bit more. I want to make sure I understand what's being said a bit better.

Nit: wrap at 100 chars (if using vscode, consider installing https://marketplace.visualstudio.com/items?itemName=stkb.rewrap).

Reply:
I misspoke a bit here. What I'm after when asking for expansion here is less "why are you doing this" and more "can you rephrase this a bit so that the unclear points I'm asking about get a bit more clear". I'm not entirely sure what the language should say here, but I know that I don't quite understand exactly this language, so I'm looking for you to help simplify that so that someone without the context of how this works would understand it.

Some parts that seem unknown here (just from reading the docs, not the source). What happens if I don't. Why is it not just handled? Isn't there some calculation that happens that excludes the end symbols? etc.

Feel free to respond in the PR comments to work out how best to phrase this, or if you're confident about what you want to say here, make changes to the PR directly (whichever works best with your workflow).